bdk_electrum_example/src/electrum.rs

```diff
@@ -0,0 +1,205 @@
+use std::{collections::BTreeMap, ops::Deref};
+
+use bdk_cli::Broadcast;
+use bdk_core::{
+    bitcoin::{BlockHash, Script, Txid},
+    sparse_chain::{InsertTxErr, SparseChain},
+    BlockId, TxHeight,
+};
+use electrum_client::{Client, Config, ElectrumApi};
+
+#[derive(Debug)]
+pub enum ElectrumError {
+    Client(electrum_client::Error),
+    Reorg(u32),
```

Comment:
I don't think this height thing is useful. What is the caller meant to do with it?

Some doc comments here would be good to explain how to recover from the two different error kinds. `Reorg` should just be call the function again.